pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -189,6 +190,8 @@ func (p *ConfigurationEditFlags) addSharedFlags(command *cobra.Command) {
 		"",
 		"Image pull secret to set. An empty argument (\"\") clears the pull secret. The referenced secret must exist in the service's namespace.")
 	p.markFlagMakesRevision("pull-secret")
+	command.Flags().Int64VarP(&p.RunAsUser, "run-as-user", "", 0, "The user to run the container (e.g., 1001).")
```

Comment:
better to clarify it's the UID, for example, `The user ID to run the container`

Reply:
Thank you. Have made the change.